packages/protocol-plugin-peer-chat/package.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "@dxos/protocol-plugin-peer-chat",
```

Comment:
Let's rename: protocol-plugin-messaging?

And later figure out how to extend for "chat" and "bot" (with actual proto messages).